Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.rst

```diff
@@ -0,0 +1,113 @@
+.. |author| replace:: *David Sanmartim*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _E-Stop-Procedure:
+
+####################
+E-Stop Procedure
+####################
+
+
+.. _E-Stop-Procedure-Overview:
+
+Overview
+========
+
+The emergency stop procedure plays a crucial role in ensuring the safety of the Vera Rubin Observatory. 
+It is typically used in emergency situations or as a safety measure. 
+It works by cutting off the electrical power to the various components of the telescope, 
+including the mount and dome, which stops them from being moved either by software commands or manually.
+
+The emergency stop can be triggered by users in certain circumstances, but it can also be activated automatically in the event of strong earthquakes 
+(currently not implemented), as well as in response to safety interlocks and software-based emergencies. 
+While the primary objective of the emergency e-stop procedure is to provide a quick and reliable means of halting the movement of the telescope and dome to avoid accidents and injuries, 
+it also serves the purpose of safeguarding the system.
+
+Below you will find a brief overview of the emergency stop procedure, covering:
+
+- The location of the relevant switches, 
+- How to activate the emergency stop, and
+- How to determine its current status.
+  
+
+.. _E-Stop-Procedure-Where-to-Find-It:
+
+Where to Find It
+================
+
+The emergency stop system in the Auxiliary Telescope can be triggered from three different locations. 
+One is located in the main ``AT Control Cabinet`` on the first floor. 
+A second one is located in the ``Dome Control Box`` on the second floor. 
+Both of these locations can be used to both engage and disengage the emergency stop.
+
+.. figure:: ./_static/e-stop-dome-control-box.jpeg
+    :name: e-stop-dome-control-box
+
+    Screenshot of the e-stop system located in the Dome Control Box.
+
+Additionally, a third emergency stop button is available upstairs in the control box used for manual movement of the dome components, 
+as illustrated in the accompanying figure. 
+It is worth noting, however, that this control box can only be used to initiate the emergency stop and cannot be used to release it, 
+as it does not have a ``Start`` button on it.
+
+.. figure:: ./_static/e-stop-up-stairs.jpeg
+    :name: e-stop-up-stairs
+
+    Screenshot of the e-stop system located upstairs.
+
+In the future, an emergency stop will also be added to the main telescope control room. 
+For now, the emergency e-stop system available in there will only work for the main telescope components.
```

Comment:
available in there --> "available there" or "available in the Control Room"  or just suggestions: 
- For now,   the e-stop button/system for the main telescope (or Simonyi Telescope? depending on the policy of documentation?) is only available in the Control Room
or
- "for now, the (physical?) e-stop system for Auxillary Telescope can be only accessed at the Auxillary Telescope site", etc.

Reply:
I have added the following sentence to make it more explicit: "The only way to activate the e-stop system for the Auxiliary Telescope is by doing it physically on site itself."